.travis.yml

```diff
@@ -39,7 +40,6 @@ addons:
         packages:
             - git
             - gnupg2
-            - realpath
```

Comment:
Why can realpath be removed now?

Reply:
The realpath command is shipped by coreutils in Ubuntu 18.04 now